public/components/settings/modules/modules.js

```diff
@@ -82,7 +120,19 @@ export class EnableModulesWrapper extends Component {
     this.setState({ extensions });
     try {
       this.currentApi && AppState.setExtensions(this.currentApi, extensions);
-    } catch (error) {} //eslint-disable-line
+    } catch (error) {
+      const options = {
+        level: UI_LOGGER_LEVELS.ERROR,
+        severity: UI_ERROR_SEVERITIES.BUSINESS,
+        error: {
```

Comment:
it is not displayed and not stored, right?

Reply:
By default is
`display: true`
`store: false`